can-define.js

```diff
@@ -719,6 +719,14 @@ define.setup = function(props, sealed) {
 };
 define.replaceWith = replaceWith;
 define.eventsProto = eventsProto;
+define.addBaseEvents = function(objPrototype){
+	objPrototype._baseEvents = objPrototype.addEventListener &&
```

Comment:
How I originally did it was something like (inside of the eventsProto addEventListener):

```js
var baseAddEventListener = this.__proto__.addEventListener && this.__proto__.addEventListener !== eventsProto.addEventListener ? this.__proto__.addEventListener : eventLifecycle.addAndSetup;

return baseAddEventListener.apply(this, arguments);
```

But it seemed wasteful to do this check every time addEventListener is called when it can be established when the type is created. But it *would* get around having to put an object on the prototype.